Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -1428,6 +1438,7 @@
 					};
 					61133B8A242393DE00786299 = {
 						CreatedOnToolsVersion = 11.3.1;
+						LastSwiftMigration = 1150;
```

Comment:
Any idea where this comes from? Do you use `Xcode 11.5`?

Reply:
no i don't know how/why that's here 🤷‍♂️ 
yes, xcode 11.5

Reply:
oh i think i know why that happened 💡 
i also have `xcode-beta` and the project was probably opened with it by default when i double-click on `xcworkspace`
because i just realized my `open by default` was changed to `xcode-beta` without any explicit action

removing this line ✅